lib/sendgrid_actionmailer.rb

```diff
@@ -188,12 +199,12 @@ def add_send_options(sendgrid_mail, mail)
          sendgrid_mail.template_id = mail['template_id'].to_s
       end
       if mail['sections']
-        json_parse(mail['sections'].value, false).each do |key, value|
+        json_parse(mail['sections'], false).each do |key, value|
```

Comment:
We might be able to not call `json_parse` at all here, although we might still need to do some key transformations to ensure the keys are still `Strings`, though.